### PR TITLE
Rework cost estimation gas prices

### PR DESCRIPTION
**Pull request by justinkaseman**
TODO:
- [ ] Rework mumbai gas cost estimation
- [ ] Add simulator estimations to `request` command

### CODE REVIEW

File: tasks/Functions-client/request.js
Comment (bolekk) [inline tasks/Functions-client/request.js:107]: wow, is that what we're supposed to do on Mumbai? Can you add a comment explaining why + a source link if any?

Comment (KuphJr) [reply tasks/Functions-client/request.js:107]: I am also curious as well.  Why does Mumbai require this different estimation?

**Review by KuphJr — APPROVED** [review]
Approving now as it appears to work just fine, but I am still curious for an explanation of the difference between networks

**Review by bolekk — CHANGES_REQUESTED** [review]
This approximation can break any time. Let's not merge it.

**Comment by bolekk** [comment]
We should debug and fix it properly, not use an arbitrary approximation.

**Pull request closed without merging**
